packages/radix-ui-themes/src/components/index.css

```diff
@@ -49,7 +49,10 @@
 @import './tooltip.css';
 
 .radix-themes:where([data-is-root-theme='true']) {
-  /* create new stacking context on the root `Theme` so layered components work out of the box */
+  /* Create a new stacking context on the root `Theme` so layered components work out of the box */
   position: relative;
   z-index: 0;
+
+  /* Make sure root `Theme` background covers the viewport */
+  min-height: 100vh;
```

Comment:
We had to add this style in every single project we used Radix Themes.

I think we shouldn't have the constraint of using Radix Themes in individual "sections" of an app; it's still going to be achievable with some friction, but the common case is simplified a ton.

Reply:
Could this potentially be a breaking change for existing users?
And have you considered using [100dvh](https://web.dev/blog/viewport-units) instead?

Reply:
Yep, we quite a few sensitive changes have snowballed, so the next public release will be 3.0.0

Will add 100dvh, forgot about it 👍